.github/workflows/main.yml

```diff
@@ -34,14 +40,16 @@ jobs:
     - name: Build with Maven
       run: mvn -B deploy --file pom.xml -Pcode-coverage,native,release -DdeployAtEnd=true
       env:
-        GITHUB_TOKEN: ${{ github.token }} 
+        MAVEN_USERNAME: ${{ secrets.OSSRH_ACCESS_ID }}
+        MAVEN_OSSRH_TOKEN: ${{ secrets.OSSRH_TOKEN }}
+        MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
     - name: Push Docker images
       run: |
-          echo '${{ secrets.GITHUB_TOKEN }}' | docker login https://docker.pkg.github.com -u '${{ github.actor }}' --password-stdin
+          echo '${{ secrets.DOCKER_TOKEN }}' | docker login https://docker.pkg.github.com -u 'nessiebuilder' --password-stdin
```

Comment:
will this show up in the build log?

Reply:
that's straight from github documentation. secrets are supposed to be masked out in job output